src/vue/backend/src/mixins/guidedSetup.js

```diff
@@ -0,0 +1,24 @@
+export const guidedSetupMixin = {
+    props: {
+        continuePath: {
+            type: String,
+            default: ''
+        },
+        skipPath: {
+            type: String,
+            default: ''
+        }
+    },
+    methods: {
+        goBackStep: function () {
+            this.$router.go(-1)
+        },
+        goSkipStep: function () {
+            this.$router.push({ path: this.skipPath })
+        },
+        async goContinue () {
+            await this.saveGeneralListingsSettings()
```

Comment:
This is too specific of an api call an cannot be reused, for now you can take this piece and abstract it to the view level. So the view would look like
```
@continue="something"

async something () {
    await this.savegeneralsettings()
    this.goContinue()
}
```